src/LogicApp/LogicApp/help/Get-AzIntegrationAccountAssembly.md

```diff
@@ -0,0 +1,159 @@
+```powershell
+PS C:\> Get-AzIntegrationAccountAssembly -ResourceGroupName "sampleResourceGroup" -IntegrationAccountName "sampleIntegrationAccount" -AssemblyName "sampleAssembly"
+```
+
+Get an assembly named "sampleAssembly" located in the integration account "sampleIntegrationAccount" which is contained in the resource group "sampleResourceGroup".
+
+### Example 2: List all assemblies in an integration account by parameters
+```powershell
+PS C:\> Get-AzIntegrationAccountAssembly -ResourceGroupName "sampleResourceGroup" -IntegrationAccountName "sampleIntegrationAccount"
+```
+
+Get all assemblies located in the integration account "sampleIntegrationAccount" which is contained in the resource group "sampleResourceGroup".
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+The integration account assembly name.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases: AssemblyName, ResourceName
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ParentName
+The integration account name.
+
+```yaml
+Type: System.String
+Parameter Sets: ByIntegrationAccount
+Aliases: IntegrationAccountName
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ParentObject
+An integration account object.
+
+```yaml
+Type: Microsoft.Azure.Management.Logic.Models.IntegrationAccount
+Parameter Sets: ByInputObject
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -ParentResourceId
+The integration account resource id.
+
+```yaml
+Type: System.String
+Parameter Sets: ByResourceId
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The resource group name.
+
+```yaml
+Type: System.String
+Parameter Sets: ByIntegrationAccount
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Microsoft.Azure.Management.Logic.Models.IntegrationAccount
```

Comment:
When you update the output/input to the PS wrapper types, be sure to use the -UpdateInputOutput parameter in Update-MarkdownHelpModule to ensure these get updated (or update manually).

Reply:
Will do. Is there ever a situation when you wouldn't want to use that flag? I have a little helper script that I run that imports everything I need and regenerates and I've always used it

Reply:
The flag exists because some teams (not ours) will add detailed description to the inputs and outputs and don't want that to be overwritten each time they regenerate the cmdlet help.  For our case though, there is not reason not to use it every time.